crates/cheatcodes/src/test/revert.rs

```diff
@@ -0,0 +1,90 @@
+use crate::{Error, Result};
```

Comment:
Is a pub(crate) module the best place for this shared logic?

Reply:
If I am not wrong the intention was to have names per cheatcodes (assume contains logic for `vm.assume*` cheatcodes, expect for `vm.expect*` cheatcodes) so revert name could be confusing in this context (though not sure what a good name could be, maybe handlers?).
If we add such probably we should move `handle_assume_no_revert` here too.

Reply:
@grandizzy Changed the file to `revert_handlers.rs` and moved the handler fns in here – lmk what you think!